core/lib/server/pub.sort.db.test.ts

```diff
@@ -0,0 +1,195 @@
+import { v4 as uuidv4 } from "uuid";
+import { describe, expect, it } from "vitest";
+
+import type { CreatePubRequestBodyWithNullsNew } from "contracts";
+import type { PubsId } from "db/public";
+import { CoreSchemaType } from "db/public";
+
+import { mockServerCode } from "~/lib/__tests__/utils";
+import { createLastModifiedBy } from "../lastModifiedBy";
+
+const { createSeed, seedCommunity } = await import("~/prisma/seed/seedCommunity");
+
+const { createForEachMockedTransaction } = await mockServerCode();
+
+const { getTrx, rollback } = createForEachMockedTransaction();
```

Comment:
I ended up moving this test to its own file, because it fails if it's run as part of another test. I think it has to do with the automatic rollback but I don't completely understand what's happening. Running the test with `it.only` or moving it to its own file make it pass, although it seems like that prevents the db from being cleaned up (including if I explicitly call rollback).

I think this test failing in a transaction makes sense, since the timestamps are all set using the transaction start time. But I'm not sure why moving it to its own file or using `it.only` prevents the transaction from working. Also is it ok to do this and just not clean up?